.github/workflows/main.yml

```diff
@@ -37,60 +37,51 @@ jobs:
           node-version: ${{ matrix.node-version }}
 
       - name: Install ocaml-lsp-server
-        run: |
-          opam pin ocaml-lsp-server https://github.com/ocaml/ocaml-lsp.git --no-action
-          opam install ocaml-lsp-server
+        run: opam install ocaml-lsp-server
```

Comment:
How does this work if there's no functional version of ocamllsp for 4.13?

Reply:
There is though: https://v3.ocaml.org/p/ocaml-lsp-server/1.9.0~4.13preview. Or is it what you meant by "functionnal"? The preview version isn't functionnal?